### ISSUE TITLE
Is HINGE PYPY compatible?

### ISSUE THREAD

**Issue by mictadlo** [HingeAssembler/HINGE#139]
Hi,
Is HINGE compatible to PYPY?

Thank you in advance.

Michal

**Comment by govinda-kamath** [comment]
I don't think so. HINGE is written primarily in C++ with some modules in python. As far as I know, that's not pypy compatible.
